assets/js/base/context/hooks/cart/use-store-cart-event-listeners.ts

```diff
@@ -36,7 +36,7 @@ const setUp = (): void => {
 const addListeners = (): void => {
 	setUp();
 
-	if ( ! window.wcBlocksStoreCartListeners.count ) {
+	if ( window.wcBlocksStoreCartListeners.count === 0 ) {
```

Comment:
This change is not directly related to this PR, but still wanted to add it. :slightly_smiling_face: 

`count` is going to be an integer, so it's more precise to compare it to `0` than to check if it's falsy.